src/components/drs/pages/DrsShow.js

```diff
@@ -1,84 +1,99 @@
 import '@fontsource/roboto';
```

Comment:
do you really need `DrsShow` and `NewDrs` anymore? The returned JSX from each is almost exactly the same (only the title appears different, which can be parameterized and added to `DrsObjectForm`).

The custom logic of updating the `activeDrsObject` based on API calls can reside in the central `Drs` class, since it holds the `activeDrsObject` in the first place